### ISSUE TITLE
Command line scripts

### ISSUE THREAD

**Issue by nilsreiter** [pkmnct/homebridge-television-universal-control#1]
Is there any chance a protocol could be „command line scripts“? I have a bunch of scripts that control my TV successfully, this function is all I need for integrating this into homekit/bridge.

**Comment by pkmnct** [comment]
@nilsreiter Yes, it is possible to execute scripts via Node.JS. I am open to pull requests to implement new protocols!
